src/index.html

```diff
@@ -1,12 +1,288 @@
+<html lang="en" class="home-page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="img">
+        <img
+          src="images/logo.png"
+          alt="moyo_logo"
+          class="img__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#Laptop" class="nav__link">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadget" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="product" data-qa="card">
+          <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="product__image"
+          >
+          <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="product__code">Product code: 195434</p>
+
+          <div class="stars-review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars-review__content">Reviews: 5</p>
+          </div>
+
+          <div class="price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
```

Comment:
```suggestion
          <div class="product__price price">
            <p class="price__text">Price:</p>
            <p class="price__value">$2,199</p>
          </div>
```
will you use it without product block?